src/settings/MappingProfiles/MappingProfiles.js

```diff
@@ -91,139 +92,86 @@ export class MappingProfiles extends Component {
     checkboxList: checkboxListShape.isRequired,
     setList: PropTypes.func.isRequired,
     showSingleResult: PropTypes.bool,
+    ENTITY_KEY: PropTypes.string,
+    EntityView: PropTypes.func,
+    EntityForm: PropTypes.func,
+    INITIAL_RESULT_COUNT: PropTypes.number,
+    RESULT_COUNT_INCREMENT: PropTypes.number,
+    actionMenuItems: PropTypes.arrayOf(PropTypes.string),
+    visibleColumns: PropTypes.arrayOf(PropTypes.string),
+    columnWidths: PropTypes.object,
+    initialValues: PropTypes.object,
   };
 
-  static defaultProps = { showSingleResult: true };
-
-  componentDidMount() {
-    this.setList();
-  }
-
-  componentDidUpdate(prevProps) {
-    const { resources } = this.props;
-
-    if (prevProps.resources !== resources) {
-      this.setList();
-    }
-  }
-
-  setList() {
-    const { setList } = this.props;
-
-    setList(this.mappingProfiles);
-  }
-
-  entityKey = ENTITY_KEYS.MAPPING_PROFILES;
-
-  visibleColumns = [
-    'selected',
-    'name',
-    'mapped',
-    'tags',
-    'updated',
-    'updatedBy',
-  ];
-
-  columnWidths = {
-    selected: 40,
-    name: 200,
-    mapped: 100,
-    tags: 150,
-    updated: 150,
-    updatedBy: 250,
+  static defaultProps = {
+    showSingleResult: true,
+    ENTITY_KEY: ENTITY_KEYS.MAPPING_PROFILES,
+    actionMenuItems: [
+      'addNew',
+      'exportSelected',
+      'selectAll',
+      'deselectAll',
+    ],
+    visibleColumns: [
+      'selected',
+      'name',
+      'mapped',
+      'tags',
+      'updated',
+      'updatedBy',
+    ],
+    columnWidths: {
+      selected: 40,
+      name: 200,
+      mapped: 100,
+      tags: 150,
+      updated: 150,
+      updatedBy: 250,
+    },
+    initialValues: {
+      name: '',
+      description: '',
+    },
+    EntityView: ViewMappingProfile,
+    // EntityForm: ActionProfilesForm,
```

Comment:
Is it needed?

Reply:
Yes

Reply:
I propose to remove then that stuff and wait for the next story.